server/services/DeckService.js

```diff
@@ -284,6 +284,9 @@ class DeckService {
             } else if (filterObject.name === 'isAlliance') {
                 filter += `AND ${this.mapColumn(filterObject.name)} = $${index++} `;
                 params.push(filterObject.value);
+            } else if (filterObject.name === 'showAllDecks' && !filterObject.value) {
```

Comment:
Please don’t change this file. It’s suppose to handle filtering on database columns only